libs/ui/src/precinct_scanner_tally_qrcode.tsx

```diff
@@ -73,8 +70,8 @@ export function PrecinctScannerTallyQrCode({
           {electionDate}, {election.county.name}, {election.state}
           <br />
           <Text small as="span">
-            Polls {pollsAction} and report created on{' '}
-            {formatFullDateTimeZone(DateTime.fromMillis(pollsToggledTime))}
+            Polls Closed and report created on{' '}
```

Comment:
I might lowercase the action here to make it read more like a sentence

Reply:
Good catch.

Reply:
Fixed